packages/calcite-components/src/components/accordion/accordion.stories.ts

```diff
@@ -119,6 +119,57 @@ export const withActions = (): string => html`
   </calcite-accordion>
 `;
 
+export const headingLevel = (): string => html`
```

Comment:
To keep snapshot count down can these tests be merged (e.g., `headingLevelAllScales` or similar)?